package.json

```diff
@@ -33,7 +33,7 @@
     "lint:src": "yarn tslint && yarn eslint-check && yarn eslint",
     "lint:src:fix": "yarn tslint --fix && yarn eslint --fix",
     "tslint": "tslint -p .",
-    "eslint": "eslint --cache --rulesdir ./eslint-rules \"./{script,eslint-rules,tslint-rules}/**/*.{j,t}s{,x}\" \"./app/{src,typings,test}/**/*.{j,t}s{,x}\" \"./changelog.json\"",
```

Comment:
If you run this rule for all `.js/.ts` files in these directories, you'll also see a `use strict` error for the emitted JS associated with our custom React rules for `tslint`:

```
/Users/shiftkey/src/desktop/eslint-rules/insecure-random.js
  1:1  error  'use strict' is unnecessary inside of modules  strict

/Users/shiftkey/src/desktop/tslint-rules/buttonGroupOrderRule.js
  1:1  error  'use strict' is unnecessary inside of modules  strict

/Users/shiftkey/src/desktop/tslint-rules/reactNoUnboundDispatcherPropsRule.js
  1:1  error  'use strict' is unnecessary inside of modules  strict

/Users/shiftkey/src/desktop/tslint-rules/reactProperLifecycleMethodsRule.js
  1:1  error  'use strict' is unnecessary inside of modules  strict

/Users/shiftkey/src/desktop/tslint-rules/reactReadonlyPropsAndStateRule.js
  1:1  error  'use strict' is unnecessary inside of modules  strict
```

`insecure-random.js` is the only non-TS module we have to worry about linting these days, so I've tweaked these rules so that the generated JS in `tslint-rules` is ignored:

 - `eslint-rules/**/*.js` 
 - `script/**/*.ts{x}`
 - `tslint-rules/**/*.ts{x}`